AdminWebsite/AdminWebsite.AcceptanceTests/Data/TestData/DefaultData.cs

```diff
@@ -9,6 +9,7 @@ public class DefaultData
         public AddParticipant AddParticipant { get; set; }
         public OtherInformation OtherInformation { get; set; }
         public Questionnaire Questionnaire { get; set; }
+        public string CancelReason { get; set; }
```

Comment:
Which page is this on?

Reply:
CancelBookingPopupComponent - the cancel popup